Source/Core/Core/IOS/USB/EmulatedUSBDevice.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
```

Comment:
Missing copyright/license header.

Reply:
With the updates migrating from a thread to using core timing, I deleted the EmulatedUSBDevice class as it was only there for the thread